cmd/launcher/desktop/desktop.go

```diff
@@ -18,12 +19,35 @@ func RunDesktop(args []string) error {
 	go exitWhenParentGone()
 	go handleSignals()
 
+	flagset := flag.NewFlagSet("launcher desktop", flag.ExitOnError)
+	var (
+		flhostname = flagset.String(
```

Comment:
Ha! I guess this _does_ need a mechanism of getting this information into desktop.

I think we really will need to tackle IPC. But in the meantime... I might have suggested environmental variables, I think they make for nicer process listings, but this is probably fine.

Reply:
agree need to tackle IPC